### PR TITLE
Policy: added health check specification

### PR DESCRIPTION
**Pull request by bryanlatten**
Also added passing test for an empty container

### CODE REVIEW

File: lib/policy.js [hunk 1]
```diff
@@ -281,6 +282,41 @@ module.exports = function() {
...
+      var noneSpecified = (healthTest && healthTest[0].toLowerCase() === 'none');
+      var noCheck = (!healthcheck || noneSpecified);
+
+      if (noCheck) {
+        msgs.addSuccess('no healthcheck specified');
+        return true;
+      }
+
+      if (isDisallowed) {
+
```
Comment (jackie) [inline lib/policy.js]: no whitespace / remove closing comment? or add whitespace and closing comment to 303-306?

**Comment by mlebrun** [comment]
👍
